css/index.css

```diff
@@ -66,16 +84,37 @@ hr {
 
 /* SOME OF THE ID SELECTORS **********************************************************/
 #top-banner {
+  position: fixed;
+  z-index: 1;
   background-color: gray;
-  padding-top: 0.9em;
-  padding-bottom: 0.3em;
+  width: 100vw;
+  font-size: 1.5em;
+  padding-top: 3%;
+  padding-bottom: 3%;
   padding-left: 0.4em;
 }
 
+#profile-pic {
+  /* set pic to stay in absolute relationship to fixed #top-banner */
+  position: absolute;
```

Comment:
do you recognize the difference between '''position: absolute; '''and '''position: relative;'''